docs/howtos/antora.yml

```diff
@@ -6,8 +6,8 @@ prerelease: true
 asciidoc:
   attributes:
     component_description: The Axon Framework how-to guides
-    type: how-to
-    group: product
+    type: howto
+    group: beginner
```

Comment:
The navigation and summary pages use "type" and "group" to know where to link the pages. Those are compared as strings. Please see the attribute of the pages in https://github.com/AxonIQ/axoniq-library-site/tree/main/content/home/modules/ROOT/pages for what the "types" are and what "groups" each type has.